code/game/objects/items/storage/backpack.dm

```diff
@@ -398,12 +389,94 @@
 	desc = "A large duffel bag for holding extra things."
 	icon_state = "duffel"
 	inhand_icon_state = "duffel"
+	actions_types = list(/datum/action/item_action/zipper)
+	storage_type = /datum/storage/duffel
+	var/zip_slowdown = 1
+	/// If this bag is zipped (contents hidden) up or not
+	/// Starts enabled so people need to figure it out to use the thing
+	var/zipped_up = TRUE
 	//slowdown = 1 //ORIGINAL
 	slowdown = 0.5 //SKYRAT EDIT CHANGE
```

Comment:
I don't think this is supposed to still be here? Unless slowdown doesn't impact zip_slowdown?